.github/workflows/benchmark.yml

```diff
@@ -17,78 +17,19 @@ concurrency:
 
 jobs:
   benchmark:
```

Comment:
I think that we should add `if: ${{ github.repository == 'dagger/dagger' }}` so that this will only run in the context of our repository.